### PR TITLE
Feature/layout details

### PR DESCRIPTION
**Pull request by hayleyw7**
# Description drawing

Please include a summary of the changes and/or which issue is fixed. Please also include relevant
motivation and context.

Many small fixes
- Fix iPhone app icon (in index.html, change href="./Images/favicon.png"/> to href="./Images/pokeball.png"/>)
- Change Router URL from /help to /howto
- Change all uses of "help" in classNames to "howto"
- Typewriter font on PokemonDetails card to match HowTo page
- Slight side margin/padding changes for PokemonDetails and HowTo

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [ ] console.log()
- [ ] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const App = () => {
         }
       />
       <Route
-        exact path='/help'
+        exact path='/howto'
```
Comment (Matt-Roden) [inline src/components/App/App.js:98]: 👍

File: src/components/HowTo/HowTo.css [hunk 2]
```diff
@@ -59,4 +58,14 @@
...
+}
+
+@media only screen and (max-width: 600px) {
+  .pokemon-details-id {
+    margin-top: 20px;
+  }   
+
+  .pokemon-details-page {
+    margin: .5rem .5rem .5rem .5rem;
+  }
```
Comment (Matt-Roden) [inline src/components/HowTo/HowTo.css:70]: awesome

**Review by Matt-Roden — APPROVED** [review]

File: src/components/App/App.js (hunk 1 above)
Comment (joanafbrito) [reply src/components/App/App.js:98]: Cool!

File: src/components/Header/Header.js [hunk 3]
```diff
@@ -9,8 +9,8 @@ const Header = ( clearPokemon ) => {
 
       <nav className='nav'>
         <Link
-          to={`/help`}
-          key={`help`}
+          to={`/howto`}
+          key={`howto`}
         >
```
Comment (joanafbrito) [inline src/components/Header/Header.js:14]: Great detail observation!

**Review by joanafbrito — APPROVED** [review]
Thank you for making the names consistent! 
Next steps:  would be a good idea also to look into the cypress files

**Pull request merged**